### PR TITLE
Prepare release Bolt 5.1.13

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

**Review by nestordedios — COMMENTED** [review]
It looks good to me but I don't understand why bumping version from `5.1.11` to `5.1.13` in some files.

**Review by xiaohutai — APPROVED** [review]
Lookin' good 👍

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "bolt",
-    "version": "5.1.11",
+    "version": "5.1.13",
```
[the remark this answers is not in the document]
Comment (bobdenotter) [reply package.json:3]: Between 5.1.11 and 5.1.12 there were no changes in JS, so no need for updated assets. Good catch. ;-)

**Review by nestordedios — APPROVED** [review]
Looks good! 👍 🚀

**Pull request merged**